### PR TITLE
control.BinaryIndex: add SourcePackage()

### PR DESCRIPTION
**Pull request by stapelberg**

### CODE REVIEW

File: control/index.go [hunk 1]
```diff
@@ -84,6 +85,19 @@ func (index *BinaryIndex) GetPreDepends() dependency.Dependency {
...
+func (index *BinaryIndex) SourcePackage() string {
+	if index.Source == "" {
+		return index.Package
+	}
+	if !strings.Contains(index.Source, " ") {
+		return index.Source
+	}
+	return strings.Split(index.Source, " ")[0]
+}
+
```
Comment (paultag) [inline control/index.go]: Hah, I've written this like 3 times now -- good call on bringing it in!

**Comment by paultag** [comment]
Has one merge conflict, otherwise LGTM, and It'll let me delete a few lines of code!

**Comment by stapelberg** [comment]
Now resolved :)

**Comment by paultag** [comment]
Thanks, man :D

**Pull request merged**